src/webui/api/synccontroller.cpp

```diff
@@ -112,6 +112,20 @@ namespace
     void processList(QVariantList prevData, const QVariantList &data, QVariantList &syncData, QVariantList &removedItems);
     QVariantMap generateSyncData(int acceptedResponseId, const QVariantMap &data, QVariantMap &lastAcceptedData, QVariantMap &lastData);
 
+    // TODO: copied from transferlistfilterswidget.cpp, move me to somewhere else?
+    QString getHost(const QString &tracker)
```

Comment:
It should move to in base/utils/net.cpp.
You must also update transferlistfilterswidget.cpp too.